.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx lint-staged
+TERM=dumb npx lint-staged
```

Comment:
I may not have searched google hard enough, but it seems like this just disables color output right? Does that really choke on big changes?

Reply:
It disables the ansi control code which seems to upset vscode